git-extra/git-update-git-for-windows

```diff
@@ -190,13 +205,22 @@ update_git_for_windows () {
 		;;
 	esac
 
-	latest=${latest_tag#v}
+	eval "$latest_eval"
 	# Did we ask about this version already?
 	recently_seen="$(get_recently_seen)"
 	test -n "$quiet" && test "x$recently_seen" = "x$latest" && return
 
 	version=$(git --version | sed "s/git version //")
-	echo "Git for Windows $version (${bit}bit)" >&2
+	version_string="Git for Windows $version (${bit}bit)"
+	filename_for_download="$bit\-bit\.exe"
+
+	if [ -d "/arm64/bin" ]
+	then
+		version_string="Git for Windows $version (ARM64)"
+		filename_for_download="ARM64\.exe"
```

Comment:
I assumed that you want to call the ARM64 binaries something like `Git-2.31.2-ARM64.exe` - happy to change if you think there might be a better name though!

Reply:
No, I concur with you: `Git-2.31.2-ARM64.exe` sounds good.